apps/minifront/src/components/swap/swap-form/simulate-swap-result/index.tsx

```diff
@@ -31,7 +31,7 @@ export const SimulateSwapResult = () => {
       {!!traces?.length && (
         <Box label='Routes'>
           <div className='flex flex-col gap-2 overflow-auto [scrollbar-width:thin]'>
-            <div className='inline-flex w-max flex-col gap-4'>
+            <div className='inline-flex w-max min-w-full flex-col gap-4'>
```

Comment:
Fixes an issue where the routes didn't stretch across the entire span of the containing div, on very wide screens